static/app/utils/profiling/profile/importProfile.spec.tsx

```diff
@@ -167,6 +169,61 @@ describe('importProfile', () => {
     expect(imported.profiles[0]).toBeInstanceOf(JSSelfProfile);
   });
 
+  it('imports sentry sampled profile', () => {
+    const sentrySampledProfile: Profiling.SentrySampledProfile = {
+      event_id: '41fed0925670468bb0457f61a74688ec',
+      version: '1',
+      os: {
+        name: 'iOS',
+        version: '16.0',
+        build_number: '19H253',
+      },
+      device: {
+        architecture: 'arm64e',
+        is_emulator: false,
+        locale: 'en_US',
+        manufacturer: 'Apple',
+        model: 'iPhone14,3',
+      },
+      timestamp: '2022-09-01T09:45:00.000Z',
+      release: '0.1 (199)',
+      platform: 'cocoa',
+      profile: {
+        samples: [
+          {
+            stack_id: 0,
+            thread_id: '0',
+            elapsed_since_start_ns: '0',
+          },
+          {
+            stack_id: 1,
+            thread_id: '0',
+            elapsed_since_start_ns: '1000',
+          },
+        ],
+        frames: [
+          {
+            name: 'main',
+            instruction_addr: '',
+          },
+          {
+            name: 'foo',
+            instruction_addr: '',
+          },
+        ],
+        stacks: [[0], [0, 1]],
+        thread_metadata: {},
```

Comment:
I think this is optional

Reply:
yeah it is, i have it right in `profiling.d.ts` these were leftover before i aligned everything to the rust struct. i'll go through and remove any of these unnecessary fields we have in our tests